review comments addressed

nelabhotlaR opened the pull request:
You create a checkpoint for the expectation suite you create, the name should be very specific. Like comic_title_validations (refer the existing checkpoint in the main branch) - 
Raghava: provided name as description_title_validations.

You create a Python script from your checkpoint, the name should be test_name_of_your_checkpoint (refer the existing checkpoint in the main branch) - 
Raghava: great_expectations checkpoint script description_title_validation 

Modify the Python script to be a pytest runnable test(refer the existing checkpoint in the main branch) - 
Raghava: Addressed - test_run_description_title_validation
 
The PR will be deemed incomplete without ^ 3

PreedhiVivek commented:
@nelabhotlaR Thanks for addressing the review comments.
I have a couple of observations.
1. The naming can be description_validation instead of description_title_validation as we are not validating title as part of your test rather only lazy one liner descriptions.
2. The min length being 15 chars is too less. Many lazy one liners like 'This is a good article' will bypass this test with this min length. So update to an appropriate 
3. Update the pytest marker to CheckpointFriday04pm in your test file as yours would be scheduled to run at that time slot.

nelabhotlaR commented:
The naming can be description_validation instead of description_title_validation as we are not validating title as part of your test rather only lazy one liner descriptions.
Raghava: Changed the checkpoint name as mentioned.

The min length being 15 chars is too less. Many lazy one liners like 'This is a good article' will bypass this test with this min length. So update to an appropriate -
Raghava: Changed the min length checking to 90 as upto 85 characters almost all coming lazy single line where as from 90 spanning to two lines.

Update the pytest marker to CheckpointFriday04pm in your test file as yours would be scheduled to run at that time slot
Raghava: Marker updated to as mentioned..
